ccpp/data/GFS_typedefs.F90

```diff
@@ -1160,13 +1160,16 @@ module GFS_typedefs
     integer              :: imfdeepcnv_samf     = 2 !< flag for SAMF scale- & aerosol-aware mass-flux deep convection scheme
     integer              :: imfdeepcnv_gf       = 3 !< flag for scale- & aerosol-aware Grell-Freitas scheme (GSD)
     integer              :: imfdeepcnv_ntiedtke = 4 !< flag for new Tiedtke scheme (CAPS)
-    integer              :: imfdeepcnv_unified  = 5 !< flag for the unified convection scheme
+    integer              :: imfdeepcnv_c3       = 5 !< flag for the Community Convective Cloud (C3) scheme
     logical              :: hwrf_samfshal           !< flag for HWRF SAMF shalcnv scheme (HWRF)
     integer              :: isatmedmf       !< flag for scale-aware TKE-based moist edmf scheme
                                             !<     0: initial version of satmedmf (Nov. 2018)
                                             !<     1: updated version of satmedmf (as of May 2019)
     integer              :: isatmedmf_vdif  = 0 !< flag for initial version of satmedmf (Nov. 2018)
     integer              :: isatmedmf_vdifq = 1 !< flag for updated version of satmedmf (as of May 2019)
+    integer              :: ichoice         = 0 !< flag for closure of C3/GF deep convection
+    integer              :: ichoicem        = 13!< flag for closure of C3/GF mid convection
```

Comment:
I'm just curious, the naming for this and the following variable are inconsistent, but is there a purpose that one has _s and the other without an underscore?